src/gluonts/model/estimator.py

```diff
@@ -40,7 +40,10 @@ def __init__(self, lead_time: int = 0, **kwargs) -> None:
         self.lead_time = lead_time
 
     def train(
-        self, training_data: Dataset, validation_data: Optional[Dataset] = None
+        self,
+        training_data: Dataset,
+        validation_data: Optional[Dataset] = None,
+        model_init: Optional[Predictor] = None,
```

Comment:
I'm not sure what the right name would be. If we pass a `Predictor`, then `model` seems "wrong".

Reply:
Also, can/should we mark this somehow as experimental?

Reply:
You mean in the documentation, or somehow raise a warning?

Reply:
> If we pass a Predictor, then model seems "wrong"

Yes, this could be `initial_predictor` maybe

Reply:
How about something like `from_predictor`, or just `predictor`?